### ISSUE TITLE
[Suggestion] Allow for the launcher to have window / selection priority

### ISSUE THREAD

**Issue by tagptroll1** [pop-os/cosmic-launcher#182]
What I mean by that is; when I open multiple applications in succession, any of the first launched apps will grab the window-focus when it launches. This will close the cosmic-launcher, even if I am mid-typing the next app I want to open. 

Having a setting to toggle that the launcher should always have focus / be active, when its open would greatly improve the feel of it. Should still be closable with the normal ways like Esc or clicking the background

**Comment by nani8ot** [comment]
Agreed. I don't think there's even a need to have a setting to keep the current behaviour. At least I don't see a reason why the launcher should be closed without the users intent (i.e. ESC, clicking the background, or selecting an entry).

An exception is fullscreen, in which case it does make sense to close the launcher. The alternative is keeping the focus on the launcher and keeping the launcher on top of the "fullscreen" window.
